transformations/suspecting_paraphraser/README.md

```diff
@@ -1,5 +1,8 @@
 # Suspecting Paraphraser 🦎  + ⌨️ → 🐍
-This paraphraser transforms a yes/no question into a subjunctive one, which helps to add more question-specific variety to the dataset.
+This paraphraser transforms a yes/no question into a subjunctive one, which helps to add more question specific informality to the dataset. 
```

Comment:
You've explained it the phenomenon anyway but if you can link in some wiki article or ref for subjunctive it might help more interested readers dig deeper.

Reply:
I've changed this name into a more fitting one (tag question) and added a source